doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,226 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
-manually. We know this type since it is uniquely identified by the topic we subscribed to (in our case ``CounterTopic``). 
-A ``reinterpret_cast`` is used to interpret the data as a ``CounterTopic``. Note that a ``static_cast`` would also work here,
-but a ``reinterpret_cast`` is used to emphasize the data-agonstic nature of the data transmission itself. 
+## Subscriber
 
-As in the untyped case we also could use a loop to get all samples as long as they are available.
+Symmetrically a subscriber also corresponds to a topic and thus needs a service description to be constructed. As for
+publishers we distinguish between typed and untyped subscribers.
 
-A non-functional approach is also possible but more verbose.
+Once a subscriber is subscribed to some topic, it is able to receive data of the type tied to this topic. In the
+untyped case this is raw memory and the user must take care that it is interpreted in a way that is compatible to the
+data that was actually sent.
 
-```cpp
-while (keepRunning)
-    {
-        // wait for new data (either sleep and wake up periodically or by notification from the waitset)
-
-        auto result = subscriber->take();
-
-        if(!result.has_error()) 
-        {
-            auto& maybeSample = result.value();
-            if (maybeSample.has_value())
-            {
-                auto& sample = maybeSample.value();
-                void* chunk = sample.get();
-
-                //interpret and process the data
-                const CounterTopic* counter = reinterpret_cast<const CounterTopic*>(chunk);
-            }
-            else
-            {
-                // we received no data
-            }
-        } else {
-            iox::popo::ChunkReceiveResult& error = result.get_error();
-            //handle the error
-        }
-    }
-```
+When multiple publishers have offered the same topic the subscriber will receive the data of all of them (but in
+indeterminate order between different publishers).
 
-### Shutdown
+## Waitset
 
-Once we are done sending, we call ``stopOffer`` at the publisher.
+The easiest way to receive data is to periodically poll whether data is available. This is sufficient for simple use
+cases but inefficient in general, as it often leads to unnecessary latency and wake-ups without data.
 
-```cpp
-publisher.stopOffer();
-```
+The ``Waitset`` can be used to relinquish control (putting the thread to sleep) and wait for user-defined ``events``
+to occur. Here an event is associated with a condition and occurs when this condition becomes true.
+Usually, these events correspond to the availability of data at specific subscribers. This way we
+can immediately wake up when data is available and will avoid unnecessary wake-ups if no data is available.
 
-Similarly the subscriber can ``unsubscribe`` to stop receiving any data.
+It manages a set of triggers which can be activated to indicate that a corresponding event occurred which wakes
```

Comment:
Implementation detail! This is not relevant for the user. I would remove the trigger part and just state for the experts that the waitset is an implementation of the reactor pattern and is informed with a push strategy that one of the attached events occured at which it informs the user.